papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,584 @@
+Furthermore, groups are aware of directives and can properly combine the directives of their constituent operations to specify resources and quantities like walltime whether executing in parallel or serial.
+
+.. code-block:: python
+
+    from flow import FlowProject
+
+    new_group = FlowProject.make_group(name="new_group")
+
+    @new_group.with_directives(
+        {"ngpu": 2,
+         "walltime": lambda job: job.doc.hours_to_run})
+    @FlowProject.post.true("simulated")
+    @FlowProject.operation
+    def simulate(job):
+        # run simulation
+        job.doc.simulated = True
+
+    @new_group
+    @FlowProject.pre.after(simulate)
+    @FlowProject.post.true("analyzed")
+    @FlowProject.operation
+    def analzye(job):
+        # analyze simulation results
+        job.doc.analzyed = True
+
+Groups also allow for specifying multiple machine specific resources (CPU or GPU) with the same operation.
+An operation can have unique directives for each distinct group to which it belongs.
+By associating an operation's directives with respect to a specific group, groups can represent distinct compute environments, such as a local workstation or a remote supercomputing cluster.
+The below snippet shows an ``expensive_simulate`` operation which can be executed with three
+different directives depending on how it is written.
+If executed through ``cpu_env`` the operation will request 48 cores, if ``gpu_env`` 4 GPUs, if
+neither then it will request 4 cores.
+This represents the real use case where an user may want to run an operation locally (in this case
+without a group), or on a CPU or GPU focused HPC/workstation.
+
+.. code-block:: python
+
+    from flow import FlowProject
+
+    cpu_env = FlowProject.make_group(name="cpu")
+    gpu_env = FlowProject.make_group(name="gpu")
+
+    @cpu_env.with_directives({"np": 48})
+    @gpu_env.with_directives({"ngpu": 4})
+    @FlowProject.operation.with_directives({"np": 4})
+    def expensive_simulate(job):
+        # expensive simulation for running on either
+        # CPUs or GPUs
+        pass
+
+Users also frequently work with multiple jobs when performing tasks such as plotting data from all jobs in the same figure.
+Though the **signac** package has methods like ``Project.groupby``, which can generate subsets of the project that are grouped by a state point key, there has been no similar feature in **signac-flow** to allow operations to act on multiple jobs.
+The concept of aggregation provides a straightforward way for users to write and submit operations that act on arbitrary subsets of jobs in a **signac** data space.
+Just as the groups feature acts as an abstraction over operations, aggregation can be viewed as an abstraction over jobs.
```

Comment:
Do you think we should italicize all occurrences of groups and aggegation/aggregates?

Reply:
I would like to italicize the first occurrence of the terms, but I'm not sure what @csadorf has in mind.

Reply:
I would be in favor italicizing those words whenever they refer to the **name** of the associated feature rather than the general concept on all occurrences. This means that "aggregation" would be italicized in many cases, but "aggregates" would not.

Reply:
If we do this, then we should likely italicize `synced collections` too.

Reply:
I went through and made these changes in 36a8cb3a8d83867.